packages/tools/client-debugger/client-debugger-view/src/components/ContainerView.tsx

```diff
@@ -109,16 +103,17 @@ export function ClientDebugView(props: ClientDebugViewProps): React.ReactElement
 					height: "100%",
 				},
 			}}
-			className={clientDebugViewClassName}
+			className={containerViewClassName}
+			key={containerId}
```

Comment:
Is this key needed for something specific?

Reply:
Doesn't look like it, I don't remember how it got here and I agree it doesn't seem necessary. Removing.